envi/archs/arm/const.py

```diff
@@ -349,6 +347,12 @@ def instrenc(encoding, index):
 INS_ASR = 85
 INS_ROR = 85
 INS_RRX = 85
+#New commands for ARMV7
```

Comment:
lol.  we need these numbers to be unique...  i realize, that's not what's there currently.  i'll fix it later.